lang/dutch.conf

```diff
@@ -1,784 +1,393 @@
-### anarchy-dutch.conf
-### By: Wesley van Laere ( Includes08 ) & Revision by: Michael ( clrxbl / https://eth0.co)
-###
-### This is the language file for Anarchy Linux.
-### This file is sourced at the beginning of the install based of the language you select
-### If you would like to help out in translating this file into your language you can easily do so by changing the variables
-###
-### Enter the name of the translator "By:" and any notes you would like to include in above comments
-### Please save your finished translation in the following format:
-###
-### anarchy-<language-here>.conf
-###
-### You may send any completed translations to the developer for it to be included in the next release:
-###
-### deadhead3492@gmail.com
-###
-### First change this variable to the correct utf-8 locale for your language.
-### Example: en_GB.UTF-8, pt_PT.UTF-8, ru_RU.UTF-8, ro_RO.UTF-8
-
-LANG="nl_NL.UTF-8"
-
-### Important: Please be sure you are using UTF-8 encoding before beginning the translation!
-### Do not change the variable names for example intro_msg=
-### Do not remove any occurrence of \n or \n\n this tells program to create a new line and should not be changed
-### Do not remove any special characters such as $a, or quotes.
-### Do not edit the names of the functions example main_msg() or remove any function brackets: { }
-### Do not edit any variables within the text for example: /dev/$DRIVE and: $user
-###
-### NOTE: Translations for additional software / desktops can be found in: etc/anarchy.conf
-### The additional translations may be added to this translation file
-### You may translate any full words contained between the quotes ""
-### Translations begin here:
-
-### New lines to translate will be added here. Do not remove these functions
-### You may, if you wish, add lines to other functions after translation
-### -------------------------------------------------------------------------------------------------
-
-translate_this() {
-
-customized_de="Customized Desktop"
-
-customized_de_msg="Customized Desktop Environments =>"
-
-change_fn="Change-Full-Username"
-
-change_sh="Verander-Shell"
-
-change_su="Verander-Rechten"
-
-mirror_msg0="Update mirrorlist:"
-
-mirror_editor_msg="Selecteer een tekst editor om de mirrors te veranderen:"
-
-manual_mirrors="Verander MirrorList Handmatig"
-
-update_mirrors_msg="Download & Rank Nieuwe Mirrors"
-
-cancel_mirrors="Sla Mirror Update Over"
-
-mirror_cancel_msg="Ben u er zeker van dat u de mirror update wilt overslaan? \n\n $a Dit kan de snelheid beinvloeden"
-
+# dutch.conf
+
+# Original Maintainer: Deadhead (Dylan Schacht)
+# Maintainer: Erazem Kokot <contact [at] erazem [dot] eu>
```

Comment:
The maintainer here means you, the translation's maintainer. You can also remove Deadhead, since he didn't originally maintain this translation (afaik).

So the only one of the three lines should be the Maintainer: <you>